src/extra_files.c

```diff
@@ -148,9 +150,23 @@ int walk_tree(struct manifest *manifest, const char *start)
 			if (F[i].dir) { /* Start of new dir to skip */
 				skip_dir = F[i].filename;
 				skip_len = strlen(skip_dir);
-				printf("%s/\n", F[i].filename);
+				if (fix) {
+					string_or_die(&temp, "%s%s", path_prefix, F[i].filename);
+					printf("REMOVING DIR %s/\n", F[i].filename);
```

Comment:
It's not an error, but there is currently a mixture of printf/fprintf(stderr) in the code, and I'm trying to better understand the convention we established recently. I would just ignore my comments for now and we can revisit later.